aim/sdk/adapters/xgboost.py

```diff
@@ -0,0 +1,45 @@
+from xgboost.callback import TrainingCallback, CallbackContainer
+from aim.sdk.session.session import Session
+from typing import Optional
+
+class AimCallback(TrainingCallback):
+    def __init__(self, repo: Optional[str] = None,
+                 experiment: Optional[str] = None):
```

Comment:
Argument `system_tracking_interval: Optional[int] = DEFAULT_SYSTEM_TRACKING_INT` is missing

Reply:
+ `flush_frequency: int = DEFAULT_FLUSH_FREQUENCY`